shell/platform/android/org/domokit/sky/shell/SkyActivity.java

```diff
@@ -4,140 +4,11 @@
 
 package org.domokit.sky.shell;
 
-import android.app.Activity;
-import android.content.Intent;
-import android.os.Build;
-import android.os.Bundle;
-import android.view.Window;
-import android.view.WindowManager;
-import io.flutter.plugin.platform.PlatformPlugin;
-import io.flutter.view.FlutterMain;
-import io.flutter.view.FlutterView;
-import java.util.ArrayList;
-import org.chromium.base.TraceEvent;
-
+import io.flutter.app.FlutterActivity;
 
 /**
- * Base class for activities that use Sky.
+ * Use io.flutter.app.FlutterActivity instead
```

Comment:
Whatever you feel comfortable with. If it's open-ended we should probably say it's open-ended.